src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -153,6 +153,13 @@ class CrossDeviceLinkUI extends Component {
   sendSms = () => {
     if (this.props.sms.valid) {
       this.setState({sending: true})
+
+      // add a quick note that this will send a production SMS, so non-production
+      // environment users will need to amend any URLs that they receive
+      if (!process.env.TRUE_PRODUCTION) {
```

Comment:
I'm scared that the name `TRUE_PRODUCTION` might create confusion and might start using it for the wrong reasons. Could we use the `development` environment for this? At the moment the `development` environment uses the same configuration as the test config but maybe they should be separate?

Reply:
just to confirm `NODE_ENV == production` is equivalent to this, no?

Reply:
No, `NODE_ENV=production` is used in production and in staging.
We want something that targets everything _but_ production.

Reply:
So I've changed this to us having an `ENV` var that 1-to-1 maps with the name of the config block it comes from
i.e. `ENV` will equal either 'production', 'test', or 'staging'